File: app/lib/models/categoria.dart
import 'package:flutter/material.dart';

/// Uma categoria de serviço (ex: Pedreiro, Eletricista, Jardineiro).
class Categoria {
  final String id;
  final String nome;
  final IconData icone;

  const Categoria({
    required this.id,
    required this.nome,
    required this.icone,
  });
}

File: android-app/app/src/main/java/com/servicos/app/model/Categoria.kt
package com.servicos.app.model

import androidx.compose.ui.graphics.vector.ImageVector

/** Uma categoria de serviço (ex: Pedreiro, Eletricista, Jardineiro). */
data class Categoria(
    val id: String,
    val nome: String,
    val icone: ImageVector,
)
